setup.py

```diff
@@ -10,7 +10,7 @@
     packages=find_packages(),
     zip_safe=False,
     install_requires=[
-        'django>=1.7a,<1.9',
+        'django>=1.10,<1.11',
```

Comment:
I know we've moved to Django 1.10, but wondering if it's not possible to still support earlier versions of Django? I didn't look at how the url APIs have changed, so maybe it just makes sense to only support 1.10 for some reason?